lib/LedgerSMB/Scripts/setup.pm

```diff
@@ -1085,6 +1085,7 @@ sub save_user {
     try { $user->create($request->{password}); }
     catch {
         if ($_ =~ /duplicate user/i){
+           $request->{dbh}->rollback;
            $request->{notice} = $request->{_locale}->text(
                        'User already exists. Import?'
```

Comment:
If we don't roll back here, we run into problems generating/querying elements for the page to be returned.